analyses/fusion_filtering/run_fusion_merged.sh

```diff
@@ -27,39 +27,43 @@ results_path="results/"
 
 
 # fusion files before and after standardization
-arriba_file="${data_path}/pbta-fusion-arriba.tsv.gz"
-starfusion_file="${data_path}/pbta-fusion-starfusion.tsv.gz"
+arriba_file="${data_path}/fusion-arriba.tsv.gz"
+starfusion_file="${data_path}/fusion-starfusion.tsv.gz"
 standard_arriba_file="${scratch_path}/arriba.tsv"
 standard_starfusion_file="${scratch_path}/starfusion.tsv"
 
 # general filtering parameters
-artifact_filter="GTEx_Recurrent|DGD_PARALOGS|Normal|BodyMap"
+artifact_filter="GTEx|HGNC_GENEFAM|DGD_PARALOGS|Normal|BodyMap"
```

Comment:
Is this change requested?

Reply:
For this one, the GTEx_Recurrent was not found and it was giving an error message - I adopted this filter from 02 the sample script.